tests/testlogout.js

```diff
@@ -0,0 +1,25 @@
+import LoginPage from '../pages/login.page';
+import HomePage from '../pages/home.page';
+import LogoutPage from '../pages/logout.page';
+
+describe ('TestLogout', async () => {
+
+    beforeEach(function(){
```

Comment:
Remember to make this function asynchronous, as you are interacting with the network here. Make the callback function `async` and use `await` before each statement inside it.

Something like:
```javascript

beforeEach(async () => {
        await LoginPage.open('login');
        await LoginPage.loginAs('jsmith@demo.io', 'Demo123!');
});
```